### DISCUSSION TITLE
在标题中使用英文冒号的问题

### DISCUSSION THREAD

**Discussion by jialiangZ** [hexojs/hexo#5382] [Question / Answer] [answered]
![image](https://github.com/hexojs/hexo/assets/54654343/ae6b061f-8006-4aad-8be1-bd36c0bf6fb4)
如图所示，我在title中使用了一篇论文，含有英文冒号，就会出错
源代码：title: PMDRnet: A Progressive Multiscale Deformable Residual Network for Multi-Image Super-Resolution of AMSR2 Arctic Sea Ice Images
![image](https://github.com/hexojs/hexo/assets/54654343/91cd55be-d4e5-4d68-a92e-3baeea74e182)

**Comment by D-Sketon** [comment, marked as answer]
需要给title加上引号

**Comment by jialiangZ** [comment]
谢谢，完美解决

**Comment by Doublefire-Chen** [comment]
> 需要给title加上引号

@D-Sketon Thank you, it also works for me!
